CMakeLists.txt

```diff
@@ -609,6 +609,10 @@ set(LIBPARQUET_SRCS
   src/parquet/exception.cc
   src/parquet/types.cc
 
+  src/parquet/arrow/reader.cc
```

Comment:
Ah I need to remove the PARQUET_ARROW flag. I'm -0 on making it optional as the dependencies are unchanged  and it doesn't have a meaningful impact on packaging or linking.